src/drive/web/modules/viewer/barviewer.styl

```diff
@@ -0,0 +1,8 @@
+@require 'settings/z-index.styl'
+bar-height = 3rem
```

Comment:
Est-ce qu'elle est différente de : https://github.com/cozy/cozy-drive/blob/master/src/drive/styles/topbar.styl#L5 ? 

Est-ce que cette variable ne devrait pas être dans UI comme son bar-index en fait ?

Reply:
Et est-ce qu'elle est différente de celle-ci : https://github.com/cozy/cozy-ui/blob/master/stylus/objects/layouts.styl#L9 

#courage :D

Reply:
IDK, maybe UI could export a variable with the bar size, but ultimately the reference is the cozy-bar style files. I'll at least unify the 2 declarations we have in our codebase.

Reply:
En fait la seconde occurence n'était même pas utilisée…